### ISSUE TITLE
dotnet is not recognized as an internal or external cammand, operable program or batch file.

### ISSUE THREAD

**Issue by Haseeb420** [dotnet/core#6312]
Problem encountered on https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/install
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

when I run the "**dotnet**" command on the command prompt the following error has occurred.
    **"**dotnet is not recognized as an internal or external cammand, operable program, or batch file.**"**

**Comment by ManickaP** [comment]
Hi @Haseeb420,

have you installed .NET? https://docs.microsoft.com/en-us/dotnet/core/install/windows?tabs=net50

**Comment by ManickaP** [comment]
> when I Install node js and git on my pc.

But have you installed .NET? If you look into `C:\Program Files` do you see directory `dotnet` and does it contain other subdirectories and `dotnet.exe`?

**Comment by Haseeb420** [comment]
yes. It contains dotnet.exe file along with the following directories host, packs, SDK,  shared, swidtag, templates.
 dotnet only works when I uninstalled node js. It is conflicting with npm.

**Comment by ManickaP** [comment]
Then few more things to check:
- Did you close and open a new command prompt after installation?
- Is the directory where dotnet.exe is in the system PATH (`echo %path%`)?

**Comment by Haseeb420** [comment]
Yes, I reopened my command after installation. 
My problem is resolved its system path variable problem.
Thanks for the help.

**Comment by Haseeb420** [comment]
@ManickaP thanks for your help and support.
